code/game/objects/items/devices/broadcast_camera.dm

```diff
@@ -49,6 +41,17 @@
 	icon_state = "[base_icon_state]0"
 	return ..()
 
+/obj/item/broadcast_camera/attack_self(mob/user, modifiers)
```

Comment:
So you just activate it by just clicking it? And it won't turn off until you click it again?

Well I don't see you adding any safeguards as such you can turn it on and the camera will be stuck **INSIDE** of you while the camera item can be anywhere. 
This is cause activating the camera creates a camera inside of your mob, and also part of the reason why I made Initially made it dual-wield only to make sure that you cannot bug yourself this way.

Reply:
added check so camera turns off when putting in storage units

Reply:
> added check so camera turns off when putting in storage units

And dropped?